nbconvert/templates/latex/document_contents.tplx

```diff
@@ -64,7 +64,7 @@
 
 % Render markdown
 ((* block markdowncell scoped *))
-    ((( cell.source | citation2latex | strip_files_prefix | convert_pandoc('markdown', 'latex') )))
+    ((( cell.source | citation2latex | strip_files_prefix | convert_pandoc('markdown', 'json',extra_args=[]) | wrapped_convert_link | convert_pandoc('json','latex'))))
```

Comment:
Open API question - do we want to have a function which encapsulates these last three steps into one:

- pandoc to json
- apply Python filters
- pandoc from json